statistics-rest/src/main/java/org/uniprot/api/statistics/service/StatisticService.java

```diff
@@ -0,0 +1,85 @@
+package org.uniprot.api.statistics.service;
+
+import org.springframework.stereotype.Service;
+import org.uniprot.api.statistics.entity.UniprotkbStatisticsEntry;
+import org.uniprot.api.statistics.mapper.StatisticMapper;
+import org.uniprot.api.statistics.model.StatisticModuleStatisticCategory;
+import org.uniprot.api.statistics.model.StatisticModuleStatisticCategoryImpl;
+import org.uniprot.api.statistics.model.StatisticModuleStatisticType;
+import org.uniprot.api.statistics.repository.StatisticsCategoryRepository;
+import org.uniprot.api.statistics.repository.UniprotkbStatisticsEntryRepository;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class StatisticService {
+
```

Comment:
We generally create an interface with name StatisticService and then implement with name StatisticServiceImpl. But we use the StatisticService type in controller and everywhere else.

Reply:
I honestly don't see any advantage unless we plan to introduce different versions of *Service. However, I added it for consistency.